example_projects/transfer/nlb_observability_stack/s3.tf

```diff
@@ -0,0 +1,70 @@
+resource "aws_s3_bucket" "nlb_logs" {
+  bucket = var.bucket_name
+}
+
+resource "aws_s3_bucket_lifecycle_configuration" "bucket-config" {
+  bucket = aws_s3_bucket.nlb_logs.id
+
+  rule {
+    id = "short_retention_policy"
+
+    expiration {
+      days = 4
+    }
+    status = "Enabled"
+  }
+}
+
+resource "aws_s3_bucket_public_access_block" "nlb_logs" {
+  bucket = aws_s3_bucket.nlb_logs.id
+
+  block_public_acls       = true
+  block_public_policy     = true
+  ignore_public_acls      = true
+  restrict_public_buckets = true
+}
+
+
+resource "aws_s3_bucket_server_side_encryption_configuration" "nlb_logs_default_encryption" {
+  bucket = aws_s3_bucket.nlb_logs.id
+
+  rule {
+    apply_server_side_encryption_by_default {
+      sse_algorithm = "AES256"
+    }
+    bucket_key_enabled = true
+  }
+}
+
+
+resource "aws_s3_bucket_policy" "nlb_logs_policy" {
+  policy = <<POLICY
+{
```

Comment:
I recommend to use https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/iam_policy_document instead of plain text json